README.md

```diff
@@ -225,3 +225,82 @@ Example:
     ```
     pytest -s ecs/mxnet/training/test_ecs_mxnet_training.py::test_ecs_mxnet_training_dgl_cpu
     ```
+
+7. To run sagemaker Local tests, Launch a cpu or gpu instance based with latest deep learning AMI.
+    a) clone your github branch with changes and run the following commands
+       ```
+       git clone https://github.com/(github_account_id)/deep-learning-containers/
+       cd deep-learning-containers && git checkout (branch_name)
+       ```
+    b) login into the ecr repo where the new docker images build are present
```

Comment:
Would change to:

Authenticate into the ECR repository that contains the docker images to test